packages/NetteToSymfony/src/Annotation/SymfonyRoutePhpDocTagNode.php

```diff
@@ -3,18 +3,23 @@
 namespace Rector\NetteToSymfony\Annotation;
 
 use PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocChildNode;
+use PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocTagNode;
+use Symplify\BetterPhpDocParser\Attributes\Attribute\AttributeTrait;
+use Symplify\BetterPhpDocParser\Attributes\Contract\Ast\AttributeAwareNodeInterface;
 
-final class RouteTagValueNode implements PhpDocChildNode
+final class SymfonyRoutePhpDocTagNode extends PhpDocTagNode implements PhpDocChildNode, AttributeAwareNodeInterface
```

Comment:
It was this easy? Extending a class? :smile:

Reply:
Yes :D and bit of work in BetterPhpDocParser :) 
https://github.com/Symplify/Symplify/pull/1437/files